astropy/tests/runner.py

```diff
@@ -1,141 +1,135 @@
 """Implements the Astropy TestRunner which is a thin wrapper around py.test."""
-
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+import inspect
 import multiprocessing
 import os
 import shlex
 import sys
 import tempfile
 import warnings
+from collections import OrderedDict
 
 from ..config.paths import set_temp_config, set_temp_cache
 from ..extern import six
 from ..utils import wraps, find_current_module
 from ..utils.exceptions import AstropyWarning
 
+class keyword(object):
+    def __init__(self, default_value, priority=0):
+        self.default_value = default_value
+        self.priority = priority
+
+    def __call__(self, f):
+        def keyword(*args, **kwargs):
+            return f(*args, **kwargs)
+        keyword._default_value = self.default_value
+        keyword._priority = self.priority
+        keyword.__doc__ = f.__doc__
+        return keyword
```

Comment:
To make a decorator that preserves call signature and docstring, try instead using the ``wraps`` function from ``astropy.utils`` (rather than explicitly copying ``__doc__``).